Cargo.toml

```diff
@@ -1,19 +1,19 @@
 [package]
 name = "tzdb"
-version = "0.3.3"
+version = "0.4.0"
 edition = "2018"
 authors = ["René Kijewski <crates.io@k6i.de>"]
 repository = "https://github.com/Kijewski/tzdb"
 description = "Static time zone information for tz-rs"
 license = "Apache-2.0"
 keywords = ["date", "time", "timezone", "zone", "calendar"]
-categories = ["date-and-time"]
+categories = ["date-and-time", "no-std"]
 readme = "README.md"
 #rust-version = "1.55"
 
 [dependencies]
-# FIXME: using "default-features = false" is a breaking change
-tz-rs = { version = "^0.6.12", features = ["const"] }
+tz-rs = { version = "^0.6.12", default-features = false, features = ["const"] }
+utcnow = { version = "^0.1.3", default-features = false }
```

Comment:
Will `utcnow` be gated behind the `local` feature?

If yes, updating the MSRV to 1.60.0 will allow using weak cargo features so the `fallback` feature doesn't enable the `utcnow` crate unless `local` is enabled.

Reply:
`utcnow` will get it's own feature gate, most likely `"now"`. You are right, I'll increase the MSRV to 1.60, so I can use the `"utcnow?/fallback"` syntax.